hypervisor/Makefile

```diff
@@ -161,8 +161,9 @@ endif
 ifeq ($(CONFIG_DMAR_PARSE_ENABLED),y)
 C_SRCS += acpi_parser/dmar_parse.c
 endif
+C_SRCS += acpi_parser/acpi_ext.c
```

Comment:
If acpi_ext.c is only used by "generic" build, should we move this file to there?

Reply:
If all ACPI info of supported boards is locked down, we can do parse only for generic boards, but given the fact that some ACPI info like wake vector might be changed on supported boards for different BIOS version, we will enable ACPI parsing for all boards during developing stage and disable it when go FuSa.